guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -1,25 +1,17 @@
-[id="Upgrading_the_External_Database_{context}"]
+[id="Upgrading_the_External_Database{context}"]
 = Upgrading the external database
 
-You can upgrade an external database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+If your {Project} deployment uses an external database, the database will be upgraded when running the {foreman-installer} on the {Project} server.
 
 .Prerequisites
-* Create a new {EL} 9 based host for PostgreSQL server that follows the external database on {EL} 9 documentation.
-ifdef::katello,orcharhino,satellite[]
-For more information, see {InstallingServerDocURL}using-external-databases_{project-context}[Using External Databases with {Project}].
-endif::[]
-* Install PostgreSQL version 13 on the new {EL} host.
+* PostgreSQL version 13 is installed on the {EL} host.
 
 .Procedure
-. Create a backup.
-. Restore the backup on the new server.
-. If {Project} reaches the new database server via the old name, no further changes are required.
-Otherwise reconfigure {Project} to use the new name:
+. Update permissions to prepare the *postgresql-evr* extension for removal:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} \
---foreman-db-host _newpostgres.example.com_ \
---katello-candlepin-db-host _newpostgres.example.com_ \
---foreman-proxy-content-pulpcore-postgresql-host _newpostgres.example.com_
+# runuser -l postgres -c \
+"psql -d foreman -c \"UPDATE pg_extension SET extowner = (SELECT oid FROM pg_authid WHERE rolname='foreman') WHERE extname='evr';\""
 ----
+. Follow xref:upgrading_a_connected_{project-context}_server_{context}[].
```

Comment:
```suggestion
ifdef::upgrading-connected[]
 . Follow xref:upgrading_a_connected_{project-context}_server_{context}[].
 endif::[]
 ifdef::upgrading-disconnected[]
 . Follow xref:upgrading_a_disconnected_{project-context}_server_{context}[].
 endif::[]
```
Can you try this? Basically, the current xref won't work for disconnected.

Reply:
Nice, I was trying to figure out how to fix that. I'll give it a try.